is-array/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-array 1.0
+// Project: https://github.com/retrofox/is-array
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isArray;
+declare function isArray(val?: any): boolean;
```

Comment:
Also, I do question whether it really makes sense to make it optional. It doesn't seem helpful for anyone.